app/javascript/lib/validation/validate-file-content.js

```diff
@@ -212,12 +224,11 @@ function sniffDelimiter(lines, mimeType) {
  * Cap lines are like meta-information lines in other file formats
  * (e.g. VCF), but do not begin with pound signs (#).
  */
-async function validateCapFormat(table, fileType) {
+function validateCapFormat([headers, annotTypes]) {
   let issues = []
-
-  // Trim spaces and quotes
-  const headers = table[0].map(header => clean(header))
-  const annotTypes = table[1].map(type => clean(type))
+  if (!headers || !annotTypes) {
+    return [['error', 'cap:format:no-header', 'File does not have at least 2 non-empty header rows']]
```

Comment:
Terminology nit: the reason I used "cap" instead of "header rows" to refer to the first two lines in metadata and cluster files is that (IIRC) SCP documentation only calls the first row the header row, and never calls the second row a header.

So this message strikes me as a case of slight term-overloading, perhaps like "cluster" or "annotation".